test/e2e/data/infrastructure-docker/main/cluster-template-topology-autoscaler/cluster-autoscaler.yaml

```diff
@@ -29,6 +29,16 @@ spec:
             annotations:
               cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: "5"
               cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: "2"
+      machinePools:
+        - class: "default-worker"
+          name: "mp-0"
+          nodeDeletionTimeout: "30s"
+          failureDomains:
+          - fd4
+          metadata:
+            annotations:
+              cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: "3"
```

Comment:
Setting the max size to 3 here so we can guarantee something is scheduled on the MachineDeployment